macros/Cargo.toml

```diff
@@ -1,17 +1,26 @@
 [package]
-name = "num-macros"
-version = "0.1.33"
 authors = ["The Rust Project Developers"]
-license = "MIT/Apache-2.0"
-homepage = "https://github.com/rust-num/num"
-repository = "https://github.com/rust-num/num"
+description = "Numeric syntax extensions"
 documentation = "http://rust-num.github.io/num"
+homepage = "https://github.com/rust-num/num"
 keywords = ["mathematics", "numerics"]
-description = "Numeric syntax extensions"
+license = "MIT/Apache-2.0"
+name = "num-macros"
+repository = "https://github.com/rust-num/num"
+version = "0.1.33"
 
-[lib]
-name = "num_macros"
-plugin = true
+[dependencies]
+quote = "0.1.3"
+syn = "0.6.0"
```

Comment:
Don't bother trying to keep up for now. I am sprinting to get serde codegen working with syn (https://github.com/serde-rs/serde/pull/548) so there may be another break coming up. When serde is done I will make a PR here bumping the version and fixing anything I broke (although I doubt there would be anything affecting the code in this PR). Also starting with the next release I will start doing proper release notes explaining the changes.